### PR TITLE
#3059 Small fixes for code quality.

### PR DESCRIPTION
**Pull request by gautierbureau**
**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [ ] unit tests and non-regression tests were added (new model, new feature and bug fix)
- [ ] main documentation was updated (update of input/output file, 3rd party, model, repository organization, solver)
- [ ] example documentations were updated (new example in examples folder)
- [ ] the corresponding milestone was added in the ticket and in this PR
- [ ] if this PR modifies the parameters or inputs/outputs of a model/solver: the corresponding xsl was added in util/xsl and platform DB were updated
- [ ] if this PR modifies a dictionary: the corresponding french dictionary was updated

### CODE REVIEW

File: dynawo/sources/Modeler/Common/DYNModelMulti.cpp [hunk 1]
```diff
@@ -1152,11 +1152,12 @@ void ModelMulti::printVariableNames() {
   Trace::debug(Trace::variables()) << "------------------------------" << Trace::endline;
   Trace::debug(Trace::variables()) << "X variables" << Trace::endline;
   Trace::debug(Trace::variables()) << "------------------------------" << Trace::endline;
+  const std::vector<propertyContinuousVar_t>& modelYType = getYType();
   for (std::vector<boost::shared_ptr<DYN::SubModel> >::const_iterator it = subModels_.begin(); it != subModels_.end(); ++it) {
     const std::vector<std::string>& xNames = (*it)->xNames();
     for (unsigned int j = 0; j < xNames.size(); ++j) {
       std::string varName = (*it)->name() + "_" + xNames[j];
-      Trace::debug(Trace::variables()) << nVar << " " << varName << Trace::endline;
+      Trace::debug(Trace::variables()) << nVar << " " << varName << " " << propertyVar2Str(modelYType[nVar]) << Trace::endline;
```
Comment (rosiereflo) [inline dynawo/sources/Modeler/Common/DYNModelMulti.cpp:1160]: We need to keep in mind that this type is the initial one. Some variables may change their type on the fly during simulation

Comment (rosiereflo) [reply dynawo/sources/Modeler/Common/DYNModelMulti.cpp:1160]: I would propose to simply add "type at time 0" or something similar before the data

Comment (gautierbureau) [reply dynawo/sources/Modeler/Common/DYNModelMulti.cpp:1160]: I modified the section with `X variables (with initial type)` would it be ok ?

Comment (rosiereflo) [reply dynawo/sources/Modeler/Common/DYNModelMulti.cpp:1160]: yes perfect

**Comment by gautierbureau** [comment]
> nrt in public/private repos will probably need to be updated too

no it's ok

File: dynawo/sources/Modeler/Common/DYNModelMulti.cpp [hunk 2]
```diff
@@ -1150,13 +1150,14 @@ void ModelMulti::printVariableNames() {
...
-  Trace::debug(Trace::variables()) << "X variables" << Trace::endline;
+  Trace::debug(Trace::variables()) << "X variables (with initial type)" << Trace::endline;
   Trace::debug(Trace::variables()) << "------------------------------" << Trace::endline;
+  const std::vector<propertyContinuousVar_t>& modelYType = getYType();
   for (std::vector<boost::shared_ptr<DYN::SubModel> >::const_iterator it = subModels_.begin(); it != subModels_.end(); ++it) {
     const std::vector<std::string>& xNames = (*it)->xNames();
     for (unsigned int j = 0; j < xNames.size(); ++j) {
       std::string varName = (*it)->name() + "_" + xNames[j];
-      Trace::debug(Trace::variables()) << nVar << " " << varName << Trace::endline;
+      Trace::debug(Trace::variables()) << nVar << " " << varName << " " << propertyVar2Str(modelYType[nVar]) << Trace::endline;
```
Comment (rosiereflo) [inline dynawo/sources/Modeler/Common/DYNModelMulti.cpp:1160]: Do we want to keep this change as I think it's not used in the final version of the delay PR?

Comment (rosiereflo) [reply dynawo/sources/Modeler/Common/DYNModelMulti.cpp:1160]: If I remember well the issue is that when we call this the dynamic types are not initialized yet

Comment (gautierbureau) [reply dynawo/sources/Modeler/Common/DYNModelMulti.cpp]: Yes it was the issue, as it is a log file I just printed twice once before calculateIC and once after with the new info when types are available. I think we don't really care about writing it twice as it should never be called most of the time

File: dynawo/sources/Modeler/Common/DYNModelMulti.cpp [hunk 3]
```diff
@@ -1150,13 +1150,18 @@ void ModelMulti::printVariableNames() {
   }
   nVar = 0;
   Trace::debug(Trace::variables()) << "------------------------------" << Trace::endline;
-  Trace::debug(Trace::variables()) << "X variables" << Trace::endline;
+  Trace::debug(Trace::variables()) << "X variables (with initial type)" << Trace::endline;
```
Comment (rosiereflo) [inline dynawo/sources/Modeler/Common/DYNModelMulti.cpp:1153]: We should adapt the header depending on withVariableType

**Review by rosiereflo — CHANGES_REQUESTED** [review]

File: dynawo/sources/Modeler/Common/DYNModelMulti.cpp (hunk 3 above)
Comment (gautierbureau) [reply dynawo/sources/Modeler/Common/DYNModelMulti.cpp]: done

**Pull request merged**